### PR TITLE
Update winlogbeat to the newest beat system tests

### PR DESCRIPTION
**Pull request by ruflin**
Apply flake8

### CODE REVIEW

File: libbeat/tests/system/beat/beat.py [hunk 1]
```diff
@@ -29,6 +31,17 @@ def start(self):
             stderr=subprocess.STDOUT)
         return self.proc
 
+    def kill(self):
+        if sys.platform.startswith("win"):
```
Comment (andrewkroh) [inline libbeat/tests/system/beat/beat.py]: Adding this for all beats will address #599, but you also need to modify the start method for Windows. See https://github.com/elastic/beats/commit/ee446a4036aa97b96147d45225655dfb850fef3b#diff-714b8651f2c293a867918ab7def84ec0L25

Comment (andrewkroh) [inline libbeat/tests/system/beat/beat.py]: You can verify #559 is fixed by looking at the zip file containing the test outputs on AppVeyor. If there are a bunch of `.cov` files, then it's fixed. :smiley_cat:

**Comment by andrewkroh** [comment]
Aside from the one comment, LGTM.

File: libbeat/tests/system/beat/beat.py (hunk 1 above)
Comment (ruflin) [inline libbeat/tests/system/beat/beat.py]: Good you spotted this. I missed the change in the start function.

**Comment by ruflin** [comment]
Fixed and new version pushed.

**Comment by andrewkroh** [comment]
I'm ignoring the AppVeyor build failure and merging. The failure wasn't caused by these changes.

**Pull request merged**